AGS_oM/eNum/Scorer.cs

```diff
@@ -0,0 +1,61 @@
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.AGS
+{
+    /***************************************************/
+
+    [Description("Defines the scorer method to be used in the fuzzy matching methods (i.e. Extract).")]
+    public enum Scorer
+    {
+        [Description("Calculates a Levenshtein simple ratio between the strings. This indicates a measure of similarity.")]
+        DefaultRatioScorer,
+        [Description("This ratio uses a heuristic called 'best partial' for when two strings are of noticeably different lengths.")]
+        PartialRatioScorer,
+        [Description("Splits the strings into tokens and computes intersections and remainders between the tokens of the two strings.A comparison string is then " +
+    "built up and is compared using the simple ratio algorithm. Useful for strings where words appear redundantly.")]
+        TokenSetScorer,
+        [Description("Splits the strings into tokens and computes intersections and remainders between the tokens of the two strings.A comparison string is then " +
+            "built up and is compared using the simple ratio algorithm. Useful for strings where words appear redundantly.")]
+        PartialTokenSetScorer,
+        [Description("Find all alphanumeric tokens in the string and sort those tokens and then take ratio of resulting joined strings.")]
+        TokenSortScorer,
+        [Description("Find all alphanumeric tokens in the string and sort those tokens and then take ratio of resulting joined strings.")]
+        PartialTokenSortScorer,
+        [Description("Similarity ratio that attempts to determine whether one strings tokens are an abbreviation of the other strings tokens. One string must have all its " +
+            "characters in order in the other string to even be considered.")]
+        TokenAbbreviationScorer,
+        [Description("Similarity ratio that attempts to determine whether one strings tokens are an abbreviation of the other strings tokens. One string must have all its " +
+            "characters in order in the other string to even be considered.")]
+        PartialTokenAbbreviationScorer,
+        [Description("Splits longer string into tokens and takes the initialism and compares it to the shorter string.")]
+        TokenInitialismScorer,
+        [Description("Splits longer string into tokens and takes the initialism and compares it to the shorter string.")]
+        PartialTokenInitialismScorer,
+        [Description("Calculates a weighted ratio between the different algorithms for best results.")]
+        WeightedRatioScorer,
```

Comment:
It would be good to add the `DisplayText` attribute on those as they would benefit from a more readable version in the UI.